### DISCUSSION TITLE
Behaviour Chips

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1714] [Proposals]
### Goal

Avaliar a proposta de unificar o comportamento entre os componentes.

### Context

Quando temos a interação "tap" nos componentes Button, Poster Card, Snap Card, Display Media Card, entre outros. Temos uma camada de sobreposição (overlay) com o token backgroundContainerPressed. Gostaríamos que o componente Chips também apresentasse esse mesmo comportamento para criar uma consistência com os demais componentes.

### Impact

Ao adicionar essa melhoria, estaríamos padronizando o comportamento dos componentes e deixaríamos evidente de forma visual para a pessoa usuária que houve uma interação no contato com o componente.

### Discussion Questions

Seria possível implementar esse comportamento componente Chips?

### Do you have a visual proposal?

https://www.figma.com/file/Rp17Kdbaa7Wxz1ttwmBlX6/Comportamento-%22pressed%22-Chips?type=design&node-id=2705%3A6405&mode=design&t=eb8dlHAEd5gNv08i-1

**Comment by aweell** [comment]
Hey @annefranca,

Thank you for your feedback!

We have a project to gradually include hover and pressed states in the components that don't currently have: https://github.com/Telefonica/mistica-design/issues/677

In particular there's an issue that covers the chips: https://github.com/Telefonica/mistica-design/issues/1527

In order to prioritize this task can you create a JIRA ticket reelated to this task?

Thank you in advance

**Reply by annefranca** [reply]
Ticket criado https://jira.tid.es/browse/OBVIVO-2536
